CMakeLists.txt

```diff
@@ -305,6 +305,14 @@ IF(WITH_CORE)
     # (Qt 3D was introduced in 5.7 but it is not stable enough in that branch)
     # Qt 5.8 is missing some classes, https://github.com/qgis/QGIS/pull/5203#discussion_r142319862
     SET(QT_MIN_VERSION 5.9.0)
+    # on recent Debian and Ubuntu the headers from Qt3DExtras library have been removed
+    # so we ship them ourselves as a workaround
+    SET (WITH_INTERNAL_QT3DEXTRAS_HEADERS FALSE CACHE BOOL "Determines whether to use our copy of Qt3DExtras headers (needed for Debian/Ubuntu)")
```

Comment:
Maybe there's a *if not package_found(qt3dextras) then set this automatically* possibility?

Reply:
Well on other distros people may be simply missing that package and we want the internal headers just as the last resort so better to have forced switch.

Reply:
Since it's specific to ubuntu/debian please do not complicate pacakging for other systems or distros :wink:

Reply:
Yes please. Rely on cmake to find the correct paths :)